### DISCUSSION TITLE
Is there any support for newer wechat accounts?

### DISCUSSION THREAD

**Discussion by Brblol** [wechaty/wechaty#2137] [Q&A]
I've been trying to login to wechat with wechaty and I cannot because my account is new.
In 2017 someone in a comment said that this issue could be fixed in an Android puppet. Has this issue been solved yet?

**Comment by huan** [comment]
Of course we support for newer WeChat accounts!

You can try our Wechaty Puppet Service with Windows/Pad protocol instead of the Web.

Learn more about Wechaty Puppet Service from https://wechaty.js.org/docs/puppet-services/ and feel free to apply one by following steps on our website!
